app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -1313,8 +1321,13 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         }
 
         fun track(eventName: String, additionalProperties: Map<String, Any>) {
+            val properties = hashMapOf<String, Any>()
```

Comment:
This method should just call track for each one of the clients, no properties added here please

Reply:
To add to this -> I would move this check into the `TrackingClientType.kt` class.

Reply:
Noted. thanks

Reply:
Noticed there are no events without context_page on android